assets/xml/objects/object_gj.xml

```diff
@@ -0,0 +1,36 @@
+<Root>
+    <File Name="object_gj" Segment="6">
+        <!-- This is the decorative rubble around the fight arena -->
+        <DList Name="gGanonsCastleRubbleAroundArenaDL" Offset="0xDC0"/>
+        <Collision Name="gGanonsCastleRubbleAroundArenaCol" Offset="0x1B70"/>
+
+        <!-- Those four are used as the pile of rubble from where Ganondorf rises to transform into Ganon -->
+        <DList Name="gGanonsCastleRubble4DL" Offset="0x2600"/>
+        <Collision Name="gGanonsCastleRubble4Col" Offset="0x2850"/>
+
+        <DList Name="gGanonsCastleRubble5DL" Offset="0x2A40"/>
+        <Collision Name="gGanonsCastleRubble5Col" Offset="0x2D28"/>
+
+        <DList Name="gGanonsCastleRubble6DL" Offset="0x2E80"/>
+        <Collision Name="gGanonsCastleRubble6Col" Offset="0x2FE4"/>
+
+        <DList Name="gGanonsCastleRubble7DL" Offset="0x3190"/>
+        <Collision Name="gGanonsCastleRubble7Col" Offset="0x33E0"/>
+
+        <!-- This three rubbles are the ones that Ganon can destroy during the fight -->
+        <DList Name="gGanonsCastleRubble2DL" Offset="0x1D20"/>
+        <Collision Name="gGanonsCastleRubble2Col" Offset="0x1F70"/>
+
+        <DList Name="gGanonsCastleRubble3DL" Offset="0x2160"/>
+        <Collision Name="gGanonsCastleRubble3Col" Offset="0x2448"/>
+
+        <DList Name="gGanonsCastleRubbleTallDL" Offset="0x3710"/>
+        <Collision Name="gGanonsCastleRubbleTallCol" Offset="0x3AF0"/>
+
+        <!-- The following are unused -->
+        <!-- This is probably an unused spike variant of the rubbles, but somebody said it looked like a laser, so I'm keeping it someone says otherwise (also, it is probably an "ia16", but in this format looks like a laser) -->
+        <Texture Name="gGanonsCastleUnusedLaserTex" OutName="ganons_castle_unused_cool_laser_beam_unused_variant" Format="rgba32" Width="32" Height="32" Offset="0x8B20"/>
```

Comment:
this is definitely not a laser lol
please use the format that is more realistic as well

Reply:
I'm the "somebody" - sorry rofl..yeah I was mostly joking. I thought it looked like a laser, but that was completely out of context. Don't listen to me when it comes to that stuff :P

Reply:
Don't worry. At the end, it was my decision to use that name instead of a serious one.
The cool laser will live in ours memories forever :purple_heart: